packages/opentelemetry-exporter-prometheus/test/PrometheusExporter.test.ts

```diff
@@ -537,11 +535,11 @@ describe('PrometheusExporter', () => {
 
                 assert.deepStrictEqual(lines, [
                   '# HELP value_recorder a test description',
-                  '# TYPE value_recorder summary',
+                  '# TYPE value_recorder histogram',
```

Comment:
> it('should export a ValueRecorder as a summary'

but this assertion was changed to a histogram, is the assertion needed until summaries are brought back? (ex: #982) probably dependent on specification updates?)